eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/json/EclairModels.scala

```diff
@@ -246,10 +237,22 @@ case class ChannelResult(
     state: ChannelState,
     feeBaseMsat: Option[MilliSatoshis],
     feeProportionalMillionths: Option[FeeProportionalMillionths],
-    data: JsObject)
+    data: JsObject) {
+  import JsonReaders._
+  def shortChannelId: ShortChannelId = (data \ "shortChannelId").validate[ShortChannelId].get
+}
 
 // ChannelResult ends here
 
+case class CreateInvoiceResult(
+  prefix: String,
```

Comment:
This can be typed a [`LnHumanReadablePart`](https://github.com/bitcoin-s/bitcoin-s/blob/master/core/src/main/scala/org/bitcoins/core/protocol/ln/LnHumanReadablePart.scala) i think